django_opensearch_dsl/documents.py

```diff
@@ -150,9 +150,13 @@ def init_prepare(self):
 
         return preparers
 
-    def prepare(self, instance):
+    def prepare(self, instance, limit_fields=None):
```

Comment:
I think `update_fields` would be a better name (and would match Django's `Model.save` similar argument.